fix: fixes incomplete type for player api function

patrick-motard opened the pull request:
`getCurrentlyPlayingTrack` returns an empty string if no song is currently
playing. The type response of that function is incorrect, leading to
complile errors. This PR fixes the issue by correcting the type
response.

Fixes #22

adamgrieger commented on src/apis/PlayerApi.ts:
```diff
@@ -48,7 +48,7 @@ export class PlayerApi {
   getCurrentlyPlayingTrack(
     options?: GetCurrentlyPlayingTrackOptions,
-  ): Promise<CurrentlyPlaying> {
+  ): Promise<CurrentlyPlaying | String> {
```
The string type should be `string` instead of `String`. The [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/basic-types.html#about-number-string-boolean-symbol-and-object) has a note about this.

patrick-motard replied:
Oops! Fixed in https://github.com/adamgrieger/spotify-web-api-ts/pull/23/commits/59c13e4e4044e64d1ebfd72c9dbcb805f8b195b7

adamgrieger reviewed (changes requested):
I need to create a contribution guide of some sort, but can you remove the `package-lock.json` file? I used `yarn` while developing this, so there's already a `yarn`-specific lock file. I don't know if people generally commit two different types of lock files.

patrick-motard commented:
> I need to create a contribution guide of some sort, but can you remove the `package-lock.json` file? I used `yarn` while developing this, so there's already a `yarn`-specific lock file. I don't know if people generally commit two different types of lock files.

I've removed the `package-lock.json` in https://github.com/adamgrieger/spotify-web-api-ts/pull/23/commits/59c13e4e4044e64d1ebfd72c9dbcb805f8b195b7. I've also added it to `.gitignore` so that npm users don't include it in future PRs. 🥳

adamgrieger commented:
Thanks, @patrick-motard!

adamgrieger commented:
:tada: This PR is included in version 1.4.5 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/spotify-web-api-ts/v/1.4.5)
- [GitHub release](https://github.com/adamgrieger/spotify-web-api-ts/releases/tag/v1.4.5)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
